Neos.Flow/Classes/Command/DoctrineCommandController.php

```diff
@@ -64,6 +71,14 @@ public function injectSettings(array $settings)
         $this->settings = $settings['persistence'];
     }
 
+    /**
+     * @param LoggerInterface $systemLogger
+     */
+    public function injectcSystemLogger(LoggerInterface $systemLogger)
```

Comment:
The method name should not work (at least not for auto-wiring), since it is misspelt. And why not `logger` (again)?